test/modules/TestResultCoordinator/Reports/DirectMethod/DirectMethodReportGenerator.cs

```diff
@@ -262,7 +262,7 @@ async Task<DirectMethodReportGeneratorMetadata> ProcessMismatchFailureCase()
             }
             else if (NetworkControllerStatus.Enabled.Equals(networkControllerStatus))
             {
-                if (HttpStatusCode.InternalServerError.Equals(statusCode))
+                if (HttpStatusCode.NotFound.Equals(statusCode))
```

Comment:
We probably want to fail the report, though, and not fail report generation, so we should change line 282 in this file to networkOffFailure++ if it is a 500

Reply:
Oh, great call